### ISSUE TITLE
[feature] refold unfolded sheet

### ISSUE THREAD

**Issue by Anthony-Gaudino** [shaise/FreeCAD_SheetMetal#296]
It would be great if we could refold an unfolded sheet metal.
This way users could unfold, make changes, and then refold.

**Comment by shaise** [comment]
I agree this can be a nice feature.

**Comment by maxwxyz** [comment]
To add:
Maybe the unfold can become a standard feature in the same tree, not a separate object.Then you could unfold, add other features (maybe a pocket) and refold (all or single bends) in the following feature. So it will stay parametric.

**Comment by shaise** [comment]
Yes, this is a very much desired feature, but quite hard to implement without refactoring the base algorithms sheetmetal uses. Hopefully we get there as well

**Comment by TimBuildIt** [comment]
this could potentially be somewhat solved by being able to create sheetmetal parts out of the faces of solid objects, though i agree it would be pretty sweet
